### DISCUSSION TITLE
Copy metadata and add translated book to it

### DISCUSSION THREAD

**Discussion by Alina-Breland** [bookfere/Ebook-Translator-Calibre-Plugin#317] [Ideas]
Hi there!

Thanks for that nice plugin again still using it a lot!

Could it be possible to copy the book with all metadata and add the translated one into it with a new language. Else I need to do all the copying myself because my custom columns are not transfered with the epub format.

I wanted to send you with paypal money but the link in the plugin didnt worked.

nice greetings to you

**Comment by bookfere** [comment]
I want to make sure I fully understand what you're asking.

Do you want to translate the ebook metadata? If so, you can check the "Metadata translation" option in the plugin's settings to achieve that. I also noticed that you mentioned "custom columns." Do you mean the metadata you added using the Calibre metadata editor should apply to the translated ebook?

Please let me know if I got that right or if you could provide more details.

---

I appreciate your generosity. Unfortunately, the old PayPal account was deprecated for some reason. I have updated the donation link in the GitHub repository and will include it in the next version of the plugin.
